test/test_standardize_format.py

```diff
@@ -113,6 +119,65 @@ def test_standardize_malformed_laz():
     assert os.path.isfile(output_file)
 
 
+@pytest.mark.parametrize(
+    "classes_to_remove",
+    [
+        [2, 3],
+        [2, 3, 4],
+        [0, 1, 2, 3, 4, 5, 6],
+    ],
+)
+def test_remove_points_from_class(classes_to_remove):
+    input_file = os.path.join(TEST_PATH, "data/classified_laz/test_data_77050_627755_LA93_IGN69.laz")
+    output_file = os.path.join(TMP_PATH, "test_remove_points_from_class.laz")
+
+    def get_pipeline_metadata(pipeline):
+        try:
+            metadata = json.loads(pipeline.metadata)
+        except TypeError:
+            d_metadata = json.dumps(pipeline.metadata)
+            metadata = json.loads(d_metadata)
+        return metadata
+
+    # count points of class not in classes_to_remove (get the point we should have in fine)
+    pipeline = pdal.Pipeline() | pdal.Reader.las(input_file)
+    where = ""
+    for cl in classes_to_remove:
+        where += "Classification != " + str(cl)
+        if cl != classes_to_remove[-1]:
+            where += " && "
```

Comment:
tu peux l'ecrire en une ligne 
`where = ' && '.join(["CLassification != " + str(cl) for cl in classes_to_remove ])`

Reply:
Bien vu ! Merci pour le trix